src/NLog.Web.AspNetCore/LayoutRenderers/AspNetTrackingConsentLayoutRenderer.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.AspNetCore.Http.Features;
+using NLog.Config;
+using NLog.LayoutRenderers;
+using NLog.Web.Enums;
+using NLog.Web.Internal;
+using System.Text;
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// ASP.NET Tracking Consent
+    /// </summary>
+    /// <remarks>
+    /// ${aspnet-tracking-consent:Property=CanTrack}
+    /// ${aspnet-tracking-consent:Property=HasConsent}
+    /// ${aspnet-tracking-consent:Property=IsConsentNeeded}
+    /// </remarks>
+    [LayoutRenderer("aspnet-tracking-consent")]
```

Comment:
Could this be changed to `aspnet-request-tracking-consent` ?

Reply:
Complete